docs/dev/api/rdc.md

```diff
@@ -1,7 +1,7 @@
 ---
 id: rdc
 title: Real Device API Endpoints
-sidebar_label: Real Devices
+sidebar_label: Legacy API v1
```

Comment:
We cannot label all of this as Legacy API. 

This API serves a different purpose in some cases and won't be deprecated or updated, it will simply coexist with the Real Device Access API